src/common/commonutils/SshUtils.c

```diff
@@ -800,7 +808,8 @@ int ProcessSshAuditCheck(const char* name, char* value, char** reason, void* log
     }
     else if (0 == strcmp(name, g_auditEnsureSshWarningBannerIsEnabledObject))
     {
-        CheckSshWarningBanner(g_sshBannerFile, g_desiredSshWarningBannerIsEnabled ? g_desiredSshWarningBannerIsEnabled : g_sshDefaultSshBannerText, reason, log);
+        CheckSshWarningBanner(g_sshBannerFile, g_desiredSshWarningBannerIsEnabled ? g_desiredSshWarningBannerIsEnabled : g_sshDefaultSshBannerText, 
+            atoi(g_desiredPermissionsOnEtcSshSshdConfig), reason, log);

```

Comment:
is this flag configurable? Seems like we only ever set `NULL`, is it really necessary?

Reply:
Yes, it is configurable, same as everything else. It's configured for the SSH server configuration file. Same access is extended to the banner file. Same thing.